src/amo/components/StaticPages/About/index.js

```diff
@@ -0,0 +1,113 @@
+          <p>
+            {i18n.gettext('The add-ons listed here have been created by thousands of developers from our community, ranging from individual hobbyists to large corporations. All publicly listed add-ons are reviewed by a team of editors before being released. Add-ons marked as Experimental have not been reviewed and should only be installed with caution.')}
+          </p>
+        </section>
+        <section>
+          <h2>{i18n.gettext('How do I keep up with what\'s happening at AMO?')}</h2>
+          <p>
+            {i18n.gettext('There are several ways to find out the latest news from the world of add-ons:')}
+          </p>
+          <ul>
+            <li>
+              {i18n.gettext('Our')}&nbsp;
+              <a href={'http://blog.mozilla.com/addons/'}>{i18n.gettext('Add-ons Blog')}</a>&nbsp;
+              {i18n.gettext('is regularly updated with information for both add-on enthusiasts and developers.')}
+            </li>
+            <li>
+              {i18n.gettext('We often post news, tips, and tricks to our Twitter account,')}&nbsp;
+              <a href={'http://twitter.com/mozamo'}>{i18n.gettext('mozamo')}</a>
+            </li>
+            <li>
+              {i18n.gettext('Our')}&nbsp;
+              <a href={'https://discourse.mozilla-community.org/c/add-ons'}>{i18n.gettext('forums')}</a>&nbsp;
+              {i18n.gettext('are a good place to interact with the add-ons community and discuss upcoming changes to AMO.')}
+            </li>
+          </ul>
+        </section>
+        <section>
+          <h2>{i18n.gettext('This sounds great! How can I get involved?')}</h2>
+          <p>{i18n.gettext('There are plenty of ways to get involved. If you\'re on the technical side:')}
+            <ul>
+              <li>
+                <Link to={'/developers'}>{i18n.gettext('Make your own add-on')}</Link>.&nbsp;
+                {i18n.gettext('We provide free hosting and update services and can help you reach a large audience of users.')}
+              </li>
+              <li>
+                {i18n.gettext('If you have add-on development experience,')}&nbsp;
+                <a href="https://wiki.mozilla.org/Add-ons/Reviewers/Applying">{i18n.gettext('become an Add-on Reviewer')}</a>!&nbsp;
+                {i18n.gettext('Our reviewers are add-on fans with a technical background who review add-ons for code quality and stability.')}
+              </li>
+              <li>
+                {i18n.gettext('Help improve this website. It\'s open source, and you can file bugs and submit patches.')}&nbsp; <a href="https://github.com/mozilla/olympia/issues"> {i18n.gettext('GitHub')}</a>&nbsp;
+                {i18n.gettext('contains all of our current bugs, legacy bugs can still be found in Bugzilla.')}
+              </li>
+            </ul>
+          </p>
+          <p>{i18n.gettext('If you\'re interested in add-ons but not quite as technical, there are still ways to help:')}</p>
```

Comment:
You can use a template string e.g: `` `Blah` `` for these strings to avoid needing to escape the quotes. Be sure not to use any template tags though since our extraction tool will automatically fix leading indents with just the straight template tag.